views/noise-candidates.haml

```diff
@@ -8,6 +8,9 @@
     = @start_date
     and
     = @end_date
+    - if @search
```

Comment:
This is indented too far, and won't show in the case where the start date is the same as the end date.